third_party/proton/tutorials/matmul.py

```diff
@@ -295,7 +295,7 @@ def enter_autotune(args, reset_only=False):
                     return
                 proton.enter_scope("<autotune>")
 
-            def exit_autotune(args):
+            def exit_autotune(args, from_exception=False):
```

Comment:
I don't think you want to pass a default argument to this, because that implies that from_exception is not always passed.

Reply:
Fix this one